tigerpath/majors_and_certificates/majors/FIT_2018.json

```
@@ -0,0 +1,362 @@
+                  "course_list": [
+                    "THR *",
+                    "MUS *",
+                    "DAN *",
+                    "VIS *",
+                    "CWR *"
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name": "Italian",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Students who wish to concentrate in the department should possess a sound knowledge of either French or Italian by the end of their sophomore year. This can be achieved by completing the language sequence (FRE 107 or 108; ITA 107 or 108), or by demonstrating equivalent qualification. Students are also strongly encouraged to complete at least one advanced-language course (FRE 207, 207F or 208; ITA 207I, 208) prior to admission, or shortly thereafter, which can then be counted toward the concentration. FIT recommends that students complete at least one more 200-level course (such as FRE 211, 215, 217, 221, 222, 224, 225, 230, or ITA 208, 209, 220, 225) before enrolling in a 300-level course.",
+              "completed_by_semester": 4,
+              "req_list": [
+                {
+                  "name": "Standard",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "Students who wish to concentrate in the department should possess a sound knowledge of either French or Italian by the end of their sophomore year. This can be achieved by completing the language sequence (FRE 107 or 108; ITA 107 or 108), or by demonstrating equivalent qualification.",
+                  "course_list": [
+                    "ITA 107",
+                    "ITA 108"
+                  ]
+                },
+                {
+                  "name": "Advanced",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "Students are also strongly encouraged to complete at least one advanced-language course (FRE 207, 207F or 208; ITA 207I, 208) prior to admission, or shortly thereafter, which can then be counted toward the concentration.",
+                  "double_counting_allowed": true,
+                  "course_list": [
+                    "ITA 207I",

```

Do they really not allow the normal ITA 207 here?
That doesn't sound right. 
It's the same course, I believe. Just one is taught normally over the semester an the other immersively in Italy over the summer.
I'm willing to bet they meant to include both.
```suggestion
                    "ITA 207",
                    "ITA 207I",
```